cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -34,31 +34,65 @@
 
    ========================================================================== #}
 
-{% macro render( settings, options={} ) %}
-
-{% if settings.heading and settings.body and settings.image_url %}
+{% if value %}
```

Comment:
I choose to edit the templates directly instead of creating a new file because it would seem redundant to maintain two separate copies. this will lead to the `jinja2\v1\wagtail` directory eventually being deleted